src/bootstrap/bootstrap.py

```diff
@@ -315,7 +315,7 @@ def build_triple(self):
         try:
             ostype = subprocess.check_output(['uname', '-s']).strip().decode(default_encoding)
             cputype = subprocess.check_output(['uname', '-m']).strip().decode(default_encoding)
-        except (subprocess.CalledProcessError, WindowsError):
+        except subprocess.CalledProcessError:
```

Comment:
Also, as of Python 3.3, WindowsError [was merged into OSError](https://docs.python.org/3/library/exceptions.html#OSError), so this definitely seems like the right fix:

> Changed in version 3.3: EnvironmentError, IOError, WindowsError, socket.error, select.error and mmap.error have been merged into OSError, and the constructor may return a subclass.